squid-exporter/e2e/Makefile

```diff
@@ -1,8 +1,8 @@
 ARCH ?= amd64
 OS ?= linux
 
-E2ETEST_K8S_VERSION  := 1.28.7
-KIND_VERSION := 0.22.0
+E2ETEST_K8S_VERSION  := 1.30.4
+KIND_VERSION := 0.23.0
```

Comment:
The combination of kind v0.23 & kindest/node:v1.30.0 or kind v0.25.0 & kindest/node:v1.30.6 are available.
https://github.com/kubernetes-sigs/kind/releases/tag/v0.23.0
https://github.com/kubernetes-sigs/kind/releases/tag/v0.25.0